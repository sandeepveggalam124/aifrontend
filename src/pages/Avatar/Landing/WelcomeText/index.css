.profile-class{
    margin-left: 340px;
    margin-top: 30px;
}