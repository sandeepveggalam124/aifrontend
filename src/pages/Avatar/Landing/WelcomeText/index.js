import React from 'react';
import { Link} from 'react-router-dom';
import './index.css';

const WelcomeText = () => {

  return (
    <div className="bg-black d-flex flex-column justify-content-center align-items-center m-5 position-relative text-center">
        <h2>Rules:</h2>
        <p>Candidates should be aware that discriminatory behavior or
        comments will not be tolerated. 
        The interview process is based solely on qualifications and merit.
        </p>
        <p>
        These rules and regulations are intended to create a 
        professional and respectful interview environment while 
        ensuring that candidates have a clear understanding of 
        expectations during the AI HR interview process.
        </p>
        <p>
        Online interview rules: Limited chances to adjust in front 
        of AIDishaCode during HR Round. Over 10 wrong head 
        moves result in disqualification; must reappear next time.
        </p>
        <div className='ms-1'>
          <button 
          type="button" 
          className="btn rounded-4 fs-6 fw-bold btn-light"
          >
            <Link to='/interview/startCapture' className='text-decoration-none text-dark'>Start</Link>
            
          </button>
        </div>
    </div>
  )
}

export default WelcomeText