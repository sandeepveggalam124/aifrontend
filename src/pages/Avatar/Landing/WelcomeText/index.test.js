import React from 'react'
import WelcomeText from './index'
import {render} from '@testing-library/react'
import { MemoryRouter as Router} from 'react-router-dom'

describe('WlecomeText component',()=>{
    let container
    beforeEach(()=>{
        const renderResult=render(
            <Router>
                <WelcomeText/>
            </Router>
        )
        container=renderResult.container
    })
    it('renders correctly',()=>{
        expect(container).toMatchSnapshot();

    })
})