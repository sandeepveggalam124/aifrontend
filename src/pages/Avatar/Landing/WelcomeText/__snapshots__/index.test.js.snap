// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WlecomeText component renders correctly 1`] = `
<div>
  <div
    class="bg-black d-flex flex-column justify-content-center align-items-center m-5 position-relative text-center"
  >
    <h2>
      Rules:
    </h2>
    <p>
      Candidates should be aware that discriminatory behavior or comments will not be tolerated. The interview process is based solely on qualifications and merit.
    </p>
    <p>
      These rules and regulations are intended to create a professional and respectful interview environment while ensuring that candidates have a clear understanding of expectations during the AI HR interview process.
    </p>
    <p>
      Online interview rules: Limited chances to adjust in front of AIDishaCode during HR Round. Over 10 wrong head moves result in disqualification; must reappear next time.
    </p>
    <div
      class="ms-1"
    >
      <button
        class="btn rounded-4 fs-6 fw-bold btn-light"
        type="button"
      >
        <a
          class="text-decoration-none text-dark"
          href="/interview/startCapture"
        >
          Start
        </a>
      </button>
    </div>
  </div>
</div>
`;
